Solutions/Azure Active Directory/Analytic Rules/NRT_MFARejectedbyUser.yaml

```diff
@@ -22,6 +28,36 @@ query: |
   | where additionalDetails_ =~ "MFA denied; user declined the authentication"
   | summarize StartTime = min(TimeGenerated), EndTIme = max(TimeGenerated) by UserPrincipalName, UserId, AADTenantId, IPAddress
   | extend Name = tostring(split(UserPrincipalName,'@',0)[0]), UPNSuffix = tostring(split(UserPrincipalName,'@',1)[0])
+  | join kind=leftouter (
```

Comment:
Unfortunately we won't be able to update this NRT query - there are limitations on this type of query that mean we can do joins or unions. We will have to skip this one and just leave it as it - https://learn.microsoft.com/en-US/azure/sentinel/near-real-time-rules#considerations

Reply:
I got rid of the changes for this one!